challenges/Back-End Challenge/api/routes/data.js

```diff
@@ -0,0 +1,147 @@
+const express = require('express');
+
+// a way to package routes together under one name
+const router = express.Router();
+
+// variable so that we can access the same list that is submitted
+let numbers;
+
+// this is to sort the list numerically instead of alphabetically
+function numberSort(a, b) {
+    return a - b;
+}
+
+//even though it should be: 
+        //router.get('/data')
+// this will be packaged as data and the base URL for every URL inside this file
+// will be /data. Therefore any route inside will be appending to /data
+
+
+
+
+// get request to resturn the current list
+router.get('/', (req, res, next) => {
+
+    // checks if the list has been created
+    if (typeof numbers !== 'undefined') {
```

Comment:
Ah! Yeah Numbers by default is simply undefined, I didn't want anyone trying to request the array when it didn't exist. Luckily it would never be able to be an empty array, so I didn't want to add checking for that as well.